manager/src/main/scala/org/openinfralabs/caerus/cache/manager/BasicStorageIOMultiTierPlanner.scala

```diff
@@ -123,96 +124,82 @@ case class BasicStorageIOPlanner(optimizer: Optimizer, predictor: Predictor, pat
           insertCaerusWrite(plan.children(i), backupPlan.children(i), caerusWrite, caerusDelete))
         plan.withNewChildren(newChildren)
     }
-  }
+  }*/
 
   override def optimize(
     plan: CaerusPlan,
-    contents: Map[Candidate,String],
+    all_contents: Map[Tier, Map[Candidate,String]],
```

Comment:
I would propose not to use underscore for names. Instead consider "allContents".

Reply:
sure